dockerfiles/codenvy/Dockerfile

```diff
@@ -22,4 +22,6 @@ RUN set -x \
     > /usr/bin/docker; chmod +x /usr/bin/docker
 ADD assembly/onpremises-ide-packaging-tomcat-codenvy-allinone/target/onpremises-ide-packaging-tomcat-codenvy-allinone-*.zip /
 RUN unzip -q onpremises-ide-packaging-tomcat-codenvy-allinone-*.zip -d /opt/codenvy-tomcat && rm onpremises-ide-packaging-tomcat-codenvy-allinone-*.zip
-CMD /opt/codenvy-tomcat/bin/catalina.sh ${JPDA} run
+COPY dockerfiles/codenvy/entrypoint.sh /
+RUN chmod +x entrypoint.sh
```

Comment:
why do we need to perform a chmod and file permission 
31  dockerfiles/alpine_jdk/entrypoint.sh → dockerfiles/codenvy/entrypoint.sh 100755 → 100644
is being changed